README.adoc

```diff
@@ -29,20 +29,31 @@ Learn how to map Java objects to database tables and perform CRUD operations usi
 
 == What you'll learn
 
-You will learn how to use the Java Persistence API (JPA) to map Java objects to database tables.
-JPA is a standard for object relational mapping (ORM) and eliminates the burden of having to write
-SQL code when performing database operations.
+You will learn how to use the Java Persistence API (JPA) to map Java objects to database tables and
+perform create, read, update and delete (CRUD) operations. 
+
+JPA is a Java specification for persisting Java objects to a relational database. The JPA specification
+is implemented by several different persistence providers. In this guide you will use EclipseLink as
+the persistence provider.
```

Comment:
Maybe clarify that Eclipselink is the JPA Provider shipped with Liberty, just so the user knows that it isn't a randomly chosen JPA Persistence Provider Implementation here.